consumer/tests/ckan/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # ----------------------------------------------------
 
 # See CKAN docs on installation from Docker Compose on usage
-FROM debian:stretch
+FROM debian:buster
```

Comment:
One stupid thing, if you are using the uploaded image... why is this here?

Reply:
From the viderum setup, dockerfile configures the image. Its usage is unclear right now. It will be streamlined to our needs after the consumer job is defined and implemented. https://jira.ehealthafrica.org/browse/AET-595

Reply:
_lost_ :face_with_head_bandage:

Reply:
or was this done by us? https://github.com/eHealthAfrica/aether-bootstrap/blob/master/ckan/Dockerfile

Reply:
I think this is a question for @shawnsarwar

Reply:
@obdulia-losantos it is necessary to setup the datastore and datapusher https://docs.ckan.org/en/2.8/maintaining/installing/install-from-docker-compose.html#datastore-and-datapusher

Reply:
:woman_shrugging: